cves/kernel/CVE-2016-2184.yml

```diff
@@ -133,10 +140,14 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
+  code: false
+  code_answer: 
+    added a check to make sure that the number of endpoints is greater than 1
+    then taking action based on that 
   fix:
   fix_answer:
+    The fix was adding a check to make sure that the number of endpoints was
+    greater than 1
```

Comment:
`fix` is missing its Boolean value. Also the notes in this section should describe if unit tests existed for this code before and after the fix, not the fix itself.